packages/config/src/chains.ts

```diff
@@ -1261,6 +1261,44 @@ export const peaqAlphanet = new Parachain({
   ws: 'wss://moonbeam.peaq.network',
 });
 
+export const peaqEvmAlphanet = new EvmParachain({
+  assetsData: [
+    {
+      // FIXME: not sure
+      asset: agng,
+      // id: 0,
```

Comment:
does this need to be fixed? is the `id` necessary?

Reply:
removed